src/main/resources/org/jpeek/xsd/skeleton.xsd

```diff
@@ -64,6 +64,15 @@ SOFTWARE.
                       </xs:sequence>
                     </xs:complexType>
                   </xs:element>
+                  <!--
+                   @todo #106:30min Refactor <ops>.
```

Comment:
@llorllale Why does the ``<method_calls>`` element have to be inside ``<ops>`` -- what is ops, actually? To me it seems better if it is inside element ``<method>``, as you did it now.

Reply:
`method/ops` contains all operations of the method, which includes calls to methods.

Reply:
@llorllale I agree with @amihaiemil , this design looks weird. Let's go with this:

```xml
<ops>
  <op code="get">attr1</op>
  <op code="set">attr2</op>
  <op code="call">the_method_name</op> <!-- this is the new "code" we are introducing -->
</ops>
```

Reply:
@yegor256 your proposal is missing the fqn of the class that the method belongs to. Where does it go?